glade/prefs.ui

```diff
@@ -497,6 +497,30 @@ along with ddterm GNOME Shell extension.  If not, see <http://www.gnu.org/licens
         <col id="2">0</col>
         <col id="3">0</col>
       </row>
+      <row>
+        <col id="0">shortcut-window-inc-height</col>
+        <col id="1" translatable="yes">Increase Window Height</col>
+        <col id="2">0</col>
+        <col id="3">0</col>
+      </row>
+      <row>
+        <col id="0">shortcut-window-dec-height</col>
+        <col id="1" translatable="yes">Decrease Window Height</col>
+        <col id="2">0</col>
+        <col id="3">0</col>
+      </row>
+      <row>
+        <col id="0">shortcut-window-inc-opacity</col>
+        <col id="1" translatable="yes">Increase Window Opacity</col>
```

Comment:
Another nitpick at action/settings names: `inc`/`dec` should probably be a prefix/suffix (i. e. `shortcut-inc-window-size`, `shortcut-dec-background-opacity`; or `shortcut-window-size-inc`, `shortcut-background-opacity-dec`), instead of being inserted in the middle of the affected setting name.

Reply:
Good points, just saw the issue with the maximized state. Added it in latest commit.